Snakefile

```diff
@@ -125,85 +129,73 @@ def clock_rate_std_dev(w):
-        metadata = "results/metadata_{subtype}_{segment}.tsv"
+        sequences = "data/sequences_{subtype}_{segment}.fasta",
     params:
-        fasta_fields =  "strain virus isolate_id date region country division location host domestic_status subtype originating_lab submitting_lab authors PMID gisaid_clade h5_clade",
-        prettify_fields = "region country division location host originating_lab submitting_lab authors PMID"
+        subtypes=subtypes_by_subtype_wildcard,
     shell:
         """
-        augur parse \
+        augur filter \
             --sequences {input.sequences} \
-            --output-sequences {output.sequences} \
-            --output-metadata {output.metadata} \
-            --fields {params.fasta_fields} \
-            --prettify-fields {params.prettify_fields}
+            --metadata {input.metadata} \
+            --query "subtype in {params.subtypes!r}" \
+            --output-sequences {output.sequences}
         """
 
-rule add_h5_clade:
-    message: "Adding in a column for h5 clade numbering"
+rule filter_metadata_by_subtype:
     input:
-        metadata = rules.parse.output.metadata,
-        clades_file = files.clades_file
+        metadata="data/metadata.tsv.zst",
     output:
-        metadata= "results/metadata-with-clade_{subtype}_{segment}.tsv"
+        metadata = "data/metadata_{subtype}.tsv",
+    params:
+        subtypes=subtypes_by_subtype_wildcard,
     shell:
         """
-        python clade-labeling/add-clades.py \
+        augur filter \
             --metadata {input.metadata} \
-            --output {output.metadata} \
-            --clades {input.clades_file}
+            --query "subtype in {params.subtypes!r}" \
+            --output-metadata {output.metadata}
         """
 
-rule add_segment_sequence_counts:
-    """
-    For each subtype's HA metadata file add a column "n_segments" which reports
-    how many segments have sequence data (no QC performed). This will force the
-    download & parsing of all segments for a given subtype. Note that this does
-    not currently consider the prescribed min lengths (see min_length function)
-    for each segment, but that would be a nice improvement.
-    """
+rule add_h5_clade:
+    message: "Adding in a column for h5 clade numbering"
     input:
-        segments = lambda w: expand("results/metadata-with-clade_{{subtype}}_{segment}.tsv", segment=SEGMENTS) \
-            if w.subtype in ['h5n1', 'h5nx'] \
-            else expand("results/metadata_{{subtype}}_{segment}.tsv", segment=SEGMENTS),
-        metadata = lambda w: "results/metadata-with-clade_{subtype}_ha.tsv" \
-            if w.subtype in ['h5n1', 'h5nx'] \
-            else "results/metadata_{subtype}_ha.tsv",
+        metadata = "data/metadata_{subtype}.tsv",
+        clades_file = files.clades_file
     output:
-        metadata = "results/metadata-segments_{subtype}_ha.tsv"
+        metadata= "results/metadata-with-clade_{subtype}.tsv"

```

Comment:
I don't think this is part of the scope of this PR, but it would seem cleaner to me for this `clade-labeling/add-clades.py` script to instead just create a node data JSON with `h5_label_clade` rather than messing with the metadata file.

@lmoncla and I just had some confusion from different rules (`refine`, `traits`) asking for the metadata TSV vs the metadata-with-clade TSV.

Reply:
>  it would seem cleaner to me for this clade-labeling/add-clades.py script to instead just create a node data JSON with h5_label_clade rather than messing with the metadata file.

Agreed. The only reason I can see to _not_ do this is if we use this data in the filtering step. But we don't.

Reply:
I created an issue for this (https://github.com/nextstrain/avian-flu/issues/25), so we can follow up on this later.